src/prop-converter.js

```diff
@@ -41,16 +45,18 @@ function propConverter(ComposedComponent) {
       // child is passed, as well as if the child is falsy.
       workingProps.children = React.Children.toArray(props.children);
 
-      // FlipMove does not support stateless functional components.
-      // Check to see if any supplied components won't work.
-      // If the child doesn't have a key, it means we aren't animating it.
-      // It's allowed to be an SFC, since we ignore it.
-      const noStateless = workingProps.children.every(child =>
-         !isElementAnSFC(child) || typeof child.key === 'undefined'
-      );
-
-      if (!noStateless) {
-        console.warn(statelessFunctionalComponentSupplied());
+      if (process.env.NODE_ENV !== 'production') {
+        // FlipMove does not support stateless functional components.
+        // Check to see if any supplied components won't work.
+        // If the child doesn't have a key, it means we aren't animating it.
+        // It's allowed to be an SFC, since we ignore it.
+        const noStateless = workingProps.children.every(child =>
+           !isElementAnSFC(child) || typeof child.key === 'undefined'
+        );
+
+        if (!noStateless) {
+          console.warn(statelessFunctionalComponentSupplied());
+        }
```

Comment:
I tried it, but it's actually harder to figure out what is going on.  When it's inline, it's easy to see that the check doesn't happen in production and what is being checked.  Since we would only call this function in one place, it doesn't make much sense to make it its own function.

If you want me to do it anyway, I will.  It's your codebase.